homeassistant/components/bring/strings.json

```
@@ -102,7 +102,7 @@
       "message": "Authentication failed for {email}, check your email and password"
     },
     "notify_missing_argument_item": {
-      "message": "Failed to perform action {service}. Notification type `Urgent message` requires a value in `Article`"
+      "message": "Failed to perform action {service}. Notification type 'Urgent message' requires a value in 'Article'"
```

This was originally intended to resemble how Home Assistant displayed errors when a required field was missing, but seems I copied the wrong error string. I looked it up and this should be correct now.
 
```suggestion
      "message": "This action requires field item, please enter a valid value for item"
```

---

The friendly name for that field in the UI is "Article" while the key is `item`.
Perhaps you want to make them identical then,